geopandas/base.py

```diff
@@ -2036,6 +2036,58 @@ def normalize(self):
         """
         return _delegate_geo_method("normalize", self)
 
+    def orient_polygons(self, exterior_cw=False):
```

Comment:
The other PR and the test key below has me wondering if we should start to opt for keyword only arguments for clarity in places `series.orient_polygons(False)` is not very descriptive.

Reply:
On agenda for Thursday.